meta-oe/recipes-devtools/guider/guider_3.9.7.bb

```diff
@@ -6,17 +6,17 @@ AUTHOR = "Peace Lee <ipeace5@gmail.com>"
 LICENSE = "GPLv2+"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=2c1c00f9d3ed9e24fa69b932b7e7aff2"
 
-PV = "3.9.6+git${SRCPV}"
+PV = "3.9.7+git${SRCPV}"
 PR = "r0"
 
 SRC_URI = "git://github.com/iipeace/${BPN}"
 #SRCREV = "${AUTOREV}"
-SRCREV = "fef25c41efb9bde0614ea477d0b90bd9565ae0b4"
+SRCREV = "52aeb6f4c536865305afa51f37166bfe46fa460d"
 
 S = "${WORKDIR}/git"
 R = "${RECIPE_SYSROOT}"
 
-inherit ${@bb.utils.contains("BBFILE_COLLECTIONS", "meta-python2", "distutils", "", d)}
+inherit distutils
```

Comment:
this should be `inherit distutils3` for py3

Reply:
yes, I fixed it.